src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class `RightTriangle` should not start with an empty line. Please remove the empty line to adhere to the coding standards.